### DISCUSSION TITLE
【灌水】大伙对新的黑曜石系列战斗任务评价如何？

### DISCUSSION THREAD

**Discussion by Kamisato100** [share121/inter-knot#2145] [其他]
![芙芙遇刺](https://github.com/user-attachments/assets/f146ce07-6078-40bb-9a4e-dcc503f74bca)

这刷出来的一堆黑曜石系列任务打得我手疼，大部分关卡难度都还能接受，道中也会多多少少给些补给，但是那个新的帮派boss和黑化版鼠妹这俩关，本人50级0+1鲨+青衣初见直接被打烂了，一个是攻击欲望特别高加双人群殴，一个是超多连段快慢刀，基本上一不小心就大残或者掉人，角色脆的像纸一样，想完成S评价拿齐菲林奖励还得多凹好多次。感觉毕竟是活动而不是深渊，建议绝妹向你大姐铁道和原学习一下弱保软好文明，把石头菲林（抽卡代币）放在过关就能拿满，然后高难或者关卡挑战（比如这次的不要掉人或者限时）给点驱动盘狗粮或者几万丁尼之类的什么额外奖励以及金牌或者彩色的通关奖章，这样能照顾一下练度比较低或者手残的玩家同时不会失去高难挑战的乐趣，不知道大伙怎么想？

**Comment by GWWA** [comment]
战斗！爽！

**Comment by Dave-12138** [comment]
还没打
一口气把任务全接了，打算全接完在hdd慢慢打，还没拿到尼尼微的S所以后面的还没动
今天继续被尼尼微暴打（
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >
